UnityGLTF/Assets/UnityGLTF/Scripts/Tests/SampleModelList.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using UnityGLTF;
+
+public class SampleModelList : MonoBehaviour
+{
+	public static string LoaderFieldName => nameof(loader);
+	public static string PathRootFieldName => nameof(pathRoot);
+	public static string ManifestRelativePathFieldName => nameof(manifestRelativePath);
+	public static string ModelRelativePathFieldName => nameof(modelRelativePath);
+	public static string LoadThisFrameFieldName => nameof(loadThisFrame);
+
+	[SerializeField]
+	private GLTFComponent loader = null;
+
+	[SerializeField]
+	private string pathRoot = "http://localhost:8080/glTF-Sample-Models/2.0/";
```

Comment:
Code still works if this folder doesn't exist?

Reply:
It gives a generic error.  I've added a nice error that describes what went wrong.  See my changes to SampleModelListInspector.cs